### PR TITLE
introduce the concept of fully booked to event times

### PR DESCRIPTION
**Pull request by jamesgorrie**
__What's your story?__
#2266

### CODE REVIEW

**Comment by jamesgorrie** [comment]
In prismic:
screen shot 2018-02-27 at 10 02 10

**Comment by jamesgorrie** [comment]
screen shot 2018-02-27 at 11 33 00
screen shot 2018-02-27 at 11 20 19

**Comment by jamesgorrie** [comment]
@jennpb @Heesoomoon we could be more conversational similar to the below?
screen shot 2018-02-27 at 11 22 50


@wellcometrust/experience-dev happy for this to go in after a review, we can get the copy around this later.

**Comment by jennpb** [comment]
This solution feels workable but not necessarily ideal visually. 

- Making the status on the promo more conversational doesn't feel quite right, and potentially opens the door to longer status messages. 
- Including a greyed out button with an email contact also sends mixed messages.
- That's a lotta 'Fully booked'!

@Heesoomoon what do you think?

File: events/app/views/pages/event.njk [hunk 1]
```diff
@@ -138,6 +152,16 @@
 
           {% if event.bookingEnquiryTeam %}
             <div class="border-top-width-1 border-color-pumice {{ {s: 2} | spacingClasses({padding: ['top', 'bottom']}) }}">
+              {% if eventInfo.isCompletelySoldOut %}
+                <div class="{{- [
+                  {s:'HNM4'} | fontClasses,
+                  {s: 4} | spacingClasses({padding: ['left', 'right']})
+                ].join(' ') }} flex-inline flex--h-center btn btn--full-width-s"
+                  disabled="disabled" aria-disabled="true">
```
Comment (gestchild) [inline events/app/views/pages/event.njk]: Guess these don't hurt, but are kinda redundant on a div

**Comment by jamesgorrie** [comment]
> Making the status on the promo more conversational doesn't feel quite right, and potentially opens the door to longer status messages.

Okay, left it as @Heesoomoon and I discussed.

> Including a greyed out button with an email contact also sends mixed messages.

Removed the email icon as per screenshot below.

> That's a lotta 'Fully booked'!

We could do it that the if everything's fully booked, we only grey out the button, which means you can't do anything.

screen shot 2018-02-27 at 14 05 28

**Pull request merged**